SPECS/python-pexpect/python-pexpect.spec

```diff
@@ -68,19 +68,17 @@ python3 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
 
 %check
 export PYTHONIOENCODING=UTF-8
-# workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1914843
-# upstream: https://github.com/pexpect/pexpect/issues/669
-# There's a patch upstream that we can presumable remove this after
-# it merges and is released.
-# Thx for the suggestion Miro: https://www.spinics.net/lists/fedora-devel/msg283026.html
-echo "set enable-bracketed-paste off" > .inputrc
```

Comment:
Can we please double check:
Locally if run the tests, All tests pass on the previous spec; but with the current changes, I can observe 6 failures.
I've re-run twice and found same results :(

![image](https://user-images.githubusercontent.com/4929861/183794350-dda40096-4ff6-4dd0-9a5f-71c2b342a1b7.png)

Reply:
That's a great catch, Falak, thank you! I've only ran the build for ARM64 to check, if the issue with `spawn_uses_env` is gone and the tests were fine there. Now, when I ran it locally, I see what you mean. I've updated the PR, the change fixed the issue on my end.